fluss-server/src/main/java/com/alibaba/fluss/server/zk/ZooKeeperClient.java

```diff
@@ -307,6 +307,37 @@ public void updateLeaderAndIsr(TableBucket tableBucket, LeaderAndIsr leaderAndIs
         LOG.info("Updated {} for bucket {} in Zookeeper.", leaderAndIsr, tableBucket);
     }
 
+    public void batchUpdateLeaderAndIsr(Map<TableBucket, LeaderAndIsr> batchUpdate)
```

Comment:
Can we integrate this method into `ZookeeperClient`? It appears to be a generic method with substantial ZooKeeper-specific implementation details.